src/EFCore.PG.NodaTime/Storage/Internal/NodaTimeMappings.cs

```diff
@@ -39,13 +39,13 @@ protected override string GenerateNonNullSqlLiteral(object value)
 
     public class TimestampLocalDateTimeMapping : NpgsqlTypeMapping
     {
-        static readonly ConstructorInfo _ctorInfo1 =
+        static readonly ConstructorInfo Constructor1 =
             typeof(LocalDateTime).GetConstructor(new[] { typeof(int), typeof(int), typeof(int), typeof(int), typeof(int) });
 
-        static readonly ConstructorInfo _ctorInfo2 =
+        static readonly ConstructorInfo Constructor2 =
```

Comment:
Let's specify what constructor accepts with prefix (for example `ConstructorWithSeconds`).